qiskit/aqua/algorithms/phase_estimators/__init__.py

```diff
@@ -0,0 +1,26 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Phase Estimators."""
+
+from .phase_estimator import PhaseEstimator
+from .phase_estimator_result import PhaseEstimatorResult
+from .phase_estimation_scale import PhaseEstimationScale
+from .hamiltonian_pe import HamiltonianPE
+
+
+__all__ = [
```

Comment:
I think we mist likely need to update the algorithms __init__.py to expose things at the algorithm level like is done for the rest of the algorithm categories.